tshirts.c

```diff
@@ -17,6 +20,15 @@ int main() {
     assert(size(37) == 'S');

```

Comment:
Boundary values need to be tested too. Since there are 3 ranges minimum 2 boundary value and 3 range values can be tested.